### PR TITLE
fill some CycloneDX gaps

### PR DESCRIPTION
**Pull request by jkowalleck**
* aded some gaps of CycloneDX things
* also introduced complete split up categories, as requested by #14
* ordered tools table alphabetically

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,14 +57,20 @@ From [Wikipedia](https://en.wikipedia.org/wiki/Software_bill_of_materials):
 - [microsoft/sbom-tool](https://github.com/microsoft/sbom-tool)
 - [SwiftBOM - generate SBOMs](https://github.com/CERTCC/SBOM/tree/master/SwiftBOM)
 - [Kubernetes SBOM Tool](https://sigs.k8s.io/bom)
-- [CycloneDX BOM Examples](https://github.com/CycloneDX/bom-examples)
```
Comment (jkowalleck) [inline README.md:57]: moved up to line 50

File: README.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ From [Wikipedia](https://en.wikipedia.org/wiki/Software_bill_of_materials):
 ### Tools (and [classification](https://ntia.gov/sites/default/files/publications/ntia_sbom_tooling_taxonomy-2021mar30_0.pdf))
 |Tool|Produce|Consume|Transform|
 |----|-------|-------|---------|
+|[CycloneDX/cyclonedx-maven-plugin](https://github.com/CycloneDX/cyclonedx-maven-plugin)|:heavy_check_mark:|||
```
Comment (jkowalleck) [inline README.md:33]: added to the table, since the tool was in the list of repositories already.

File: README.md [hunk 3]
```diff
@@ -30,6 +30,8 @@ From [Wikipedia](https://en.wikipedia.org/wiki/Software_bill_of_materials):
...
+|[CycloneDX CLI tool](https://github.com/CycloneDX/cyclonedx-cli)||:heavy_check_mark:|:heavy_check_mark:|
```
Comment (jkowalleck) [inline README.md:34]: added a tool that is capable of transforming, converting and merging SBOMs

File: README.md [hunk 4]
```diff
@@ -54,14 +57,20 @@ From [Wikipedia](https://en.wikipedia.org/wiki/Software_bill_of_materials):
...
 - [Kubernetes SBOM Tool](https://sigs.k8s.io/bom)
-- [CycloneDX BOM Examples](https://github.com/CycloneDX/bom-examples)
 - [Aqua Trivy](https://github.com/aquasecurity/trivy)
 - [bomber](https://github.com/devops-kung-fu/bomber)
   - [Snyk provider](https://github.com/devops-kung-fu/bomber/tree/main/providers/snyk)
 - Snyk SBOM [API](https://docs.snyk.io/snyk-api-info) and [CLI](https://docs.snyk.io/snyk-cli)
 - [Snyk SBOM Checker](https://snyk.io/code-checker/sbom-security/)
 - [Interlynk SBOM Quality Score](https://github.com/interlynk-io/sbomqs)
 
+## CycloneDX
```
Comment (jkowalleck) [inline README.md:67]: some initial CycloneDX reads

**Comment by developer-guy** [comment]
cool, thx, can you please rebase from master?

**Comment by jkowalleck** [comment]
re: https://github.com/awesomeSBOM/awesome-sbom/pull/18#issuecomment-1445991339

done

**Review by Dentrax — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -28,23 +28,27 @@ From [Wikipedia](https://en.wikipedia.org/wiki/Software_bill_of_materials):
...
-|[microsoft/sbom-tool](https://github.com/microsoft/sbom-tool)|SPDX|||||||
-|[SwiftBOM - generate SBOMs](https://github.com/CERTCC/SBOM/tree/master/SwiftBOM)|CycloneDX,SPDX,SWID|||||
-|[Kubernetes SBOM Tool](https://sigs.k8s.io/bom)|SPDX|||||
-|[Aqua Trivy](https://github.com/aquasecurity/trivy)|CycloneDX,SPDX|CycloneDX,SPDX|CycloneDX,SPDX|||
-|[bomber](https://github.com/devops-kung-fu/bomber)||CycloneDX,SPDX|CycloneDX,SPDX||||
-|Snyk SBOM [API](https://docs.snyk.io/snyk-api-info) and [CLI](https://docs.snyk.io/snyk-cli)|CycloneDX,SPDX|||
-|[Snyk SBOM Checker](https://snyk.io/code-checker/sbom-security/)||CycloneDX,SPDX|||||
-|[Interlynk SBOM Quality Score](https://github.com/interlynk-io/sbomqs)||CycloneDX,SPDX|CycloneDX,SPDX||CycloneDX,SPDX|
+
+|Tool|Build SBOM|Analyze SBOM|Edit SBOM|View SBOM| Diff SBOM |Import SBOM|Translate SBOM|Merge SBOM|Integrate with Other Tools|
```
Comment (jkowalleck) [inline README.md:32]: complete categories as of #14

**Comment by developer-guy** [comment]
need rebase again

**Comment by jkowalleck** [comment]
@developer-guy needs review again.

**Review by developer-guy — APPROVED** [review]
LGTM, thanks!

File: README.md [hunk 6]
```diff
@@ -28,32 +28,36 @@ From [Wikipedia](https://en.wikipedia.org/wiki/Software_bill_of_materials):
...
-|[SBOM4Files](https://pypi.org/project/sbom4files/)| CycloneDX,SPDX    |||||
-|[Distro2SBOM](https://pypi.org/project/distro2sbom/)| CycloneDX,SPDX    |||||
-|[SBOMDiff](https://pypi.org/project/sbomdiff/)||CycloneDX,SPDX     |CycloneDX,SPDX||||
-|[SBOM2doc](https://pypi.org/project/sbom2doc/)|| CycloneDX,SPDX    |CycloneDX,SPDX||||
-|[SBOM2dot](https://pypi.org/project/sbom2dot/)|| CycloneDX,SPDX    |CycloneDX,SPDX||||
-|[SBOMAudit](https://pypi.org/project/sbomaudit/)|| CycloneDX,SPDX    |CycloneDX,SPDX||||
-|[SBOM-Manager](https://pypi.org/project/sbom-manager/)|| CycloneDX,SPDX    |CycloneDX,SPDX||||
+
+|Tool|Build SBOM|Analyze SBOM|Edit SBOM|View SBOM|Diff SBOM|Import SBOM|Translate SBOM|Merge SBOM|Integrate with Other Tools|
+|----|:--------:|:----------:|:-------:|:-------:|:-------:|:---------:|:------------:|:--------:|:------------------------:|
```
Comment (jkowalleck) [inline README.md:33]: - also introduced complete split up categories, as requested by https://github.com/awesomeSBOM/awesome-sbom/issues/14
- ordered the table alphabetically, so the next rebase is easier for me.

**Pull request merged**